app/views/state_file/questions/landing_page/edit.html.erb

```diff
@@ -19,17 +21,30 @@
         <%=t(".#{params[:us_state]}.supported_by") %>
       </div>
     </div>
-    <div class="blue-group spacing-below-60">
-      <%= t(".help_text_html") %>
-    </div>
 
-    <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
-      <%= f.submit t("general.get_started"), class: "button button--primary button--wide", id: "firstCta" %>
+    <% if @user_name.present? %>
+      <h2 class="h2"><%= t(".welcome_back", user_name: @user_name) %></h2>
+      <h2 class="h2"><%= t(".continue", state_name: @state_name) %></h2>
+      <%= link_to StateFile::StateFilePagesController.to_path_helper(action: :login_options, us_state: current_intake.state_code), class: "button button--primary button--wide", id: "firstCta" do %>
+        <%= t("general.sign_in") %>
+      <% end %>
+      <%= form_with model: @form, url: { action: :update }, local: true, method: :put, builder: VitaMinFormBuilder, id: "start-again-form" do |f| %>
+        <h2 class="h2"><%= t(".not_you_html",
+          new_state_return_url: "javascript:document.querySelector('#start-again-form').submit()",
+          state_name: @state_name,
+          user_name: @user_name
+        )%></h2>
```

Comment:
hm it would be nice if we could avoid having a submit button in an h2, is there a reason this can't be a button with h2 style?

Reply:
The reason I used a link was that this matches the mock - if you use a button then the inline styling becomes more complicated (Getting it to appear inline rather than as a block cross browser is tricky)